pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "requests-file"
+version = "1.5.1"
```

Comment:
Should `version` be removed, since version is inferred from `tool.setuptools_scm`? I haven't looked into setuptools_scm tbh so not sure about all the details there.